rasr/feature_scorer.py

```diff
@@ -126,25 +127,54 @@ def __init__(self, prior_mixtures, scale=1.0, priori_scale=0.0, prior_file=None)
         self.config.normalize_mixture_weights = False
 
 
-class TFLabelContextFeatureScorer(FeatureScorer):
+class OnnxFeatureScorer(FeatureScorer):
     def __init__(
         self,
-        fs_tf_config,
-        contextPriorFile,
-        diphonePriorFile,
-        prior_mixtures,
-        prior_scale,
+        *,
+        mixtures,
+        model,
+        io_map,
+        label_log_posterior_scale=1.0,
+        label_prior_scale=0.7,
+        label_log_prior_file=None,
+        apply_log_on_output=False,
+        negate_output=True,
+        intra_op_threads=1,
+        inter_op_threads=1,
+        **kwargs,
     ):
-        super().__init__()
+        """
+        :param str|tk.Path mixtures: path to a *.mix file e.g. output of either EstimateMixturesJob or CreateDummyMixturesJob
```

Comment:
@michelwi might correct me here, but I think using | for `Union` is deprecated.

Reply:
But this also just might be flavor, maybe someone like @JackTemaki knows?

Reply:
>  I think using | for Union is deprecated.

@Atticus1806 Actually it's the other way around. Using `|` is the more "modern" way. Type annotations with `|` instead of `typing.Union` were introduced in python 3.10. But since i6_core is currently based on python3.8 `Union` is correct here.

Reply:
Ahh good to know. But if 3.8 already "allows" it I would not mind keeping it like this.